de-lib/src/main/java/org/iplantc/de/theme/base/client/analyses/AnalysesMessages.properties

```diff
@@ -38,3 +38,15 @@ shareSupportMi = Share with support...
 shareSupportConfirm = Your output folder(s) and file(s) will be shared with CyVerse support team. Do you want to continue ?
 shareWithInput = Yes. Share my input files as well!
 shareOutputOnly = Share output folder(s) only!
+outputUnexpected = Analysis completed but I got unexpected output.
+noOutput = Analysis completed but I got no output.
+selectCondition = Select the condition of your analysis:
+comments = Comments
+needHelp = I still need help!
+submit = Submit
+agreeToShare = <span style="overflow:wrap;">I agree to <a href="https://wiki.cyverse.org/wiki/x/KBfHAQ" target="_blank">share my analysis</a> details, output file(s) and logs with CyVerse Support. <br/>If you want any of the <a href="https://wiki.cyverse.org/wiki/x/XRfHAQ" target="_blank">Science Informaticians</a> to recuse themselves from assisting with this problem, please list their name(s) in comments.</span>
+disclaimer = Disclaimer
+termsOfSupport = You must agree to terms of support!
+supportRequestFailed = Unable to submit your support request. Please email support at support@cyverse.org.
+supportRequestSuccess = Your support request is submitted. A support staff will contact you shortly.
+requestHelp = requesting help with Analysis
```

Comment:
"requesting" should probably capitalized.  And actually, Kathleen should probably review these strings and the information in those HTML files that were added as well.